dialogue-core/src/main/java/com/palantir/dialogue/core/DeprecationWarningChannel.java

```diff
@@ -82,9 +85,10 @@ static EndpointChannel create(Config cf, EndpointChannel delegate, Endpoint endp
             }
 
             meter.mark();
-            if (tryAcquire(endpoint.serviceName())) {
+            if (tryAcquire(channelName, endpoint)) {
                 log.warn(
```

Comment:
also would you mind renaming the SafeArg.of("service" to "server"? the log screenshot I saw in slack looked v odd